server/index.js

```diff
@@ -76,6 +78,51 @@ app.post('/api/items', (req, res, next) => {
     });
 });
 
+app.get('/api/items', (req, res, next) => {
+  const sql = `
+    select "itemId", "url", "title", "description", "price", "numInStock",
+    "Items"."createdAt", "type", "value"  from "Items" left join "Variations"
+    using ("itemId") order by "itemId";
```

Comment:
Try using a `json_agg("Variations")` so that you don't have to write a log of JavaScript code to reformat your rows.

https://www.postgresql.org/docs/12/functions-aggregate.html